.github/workflows/ci.yaml

```diff
@@ -10,5 +10,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 12.x
-      - run: npm install -g markdownlint-cli
-      - run: markdownlint ./
\ No newline at end of file
+      - name: Install node_modules
+        run: yarn
```

Comment:
Nice, does this mean that the version of markdownlint is now versioned according to devDependencies? (I'm not too familiar with how node/npm/yarn handle this)

If so, sweet!

Reply:
Yep!